ls1intum/Artemis
# Notification Sidebar

### Checklist

### Motivation and Context
Some future functionalities will be added to the notification system and to maintain clarity, the notifications should be displayed in a sidebar. This is a draft of the sidebar:

![image](https://user-images.githubusercontent.com/15168372/79138924-a7716300-7db5-11ea-8f0e-b89be2a691e1.png)

### Description
This PR prepares the future sidebar. Only existing functionality has been incorporated into the new UI.

Further refactoring of the existing code will be part of a future PR.

### Steps for Testing
1. Log in to Artemis
2. Click on the notification icon in the top navigation
3. Open/close the sidebar (cross-browser)
4. Check the pre-existing functionality (notification count, evaluation of *recent* notifications)

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/79141137-78f58700-7db9-11ea-8b91-4cf321f719e8.png)
